[ELEC2811] Add summary (#642)
Gp2mv3/Syntheses: src/q7/instrusensors-ELEC2811/summary/L1/characterization.tex

of the measurement conditions (e.g. a thermistor).
\end{mydef}

\begin{mydef}
A \textbf{relative} sensor produces a signal that relates to
specific measurement conditions (e.g. a thermocouple).
A thermocouple output signal cannot be related to any particular 
temperature without referencing to a known baseline. 
\end{mydef}

Another example of the absolute and relative sensors is a pressure sensor. An absolute pressure sensor produces signal in reference to vacuum (absolute zero on a pressure scale). A relative pressure sensor produces signal with respect to a selected baseline that is not zero pressure (e.g. to the atmospheric pressure).
\vspace{0.3cm}

A photodiode is an \textbf{absolute active} sensor as the generated photocurrent varies with the absolute illumination level. 

\section{Characterization}

\subsection{Transfer function}

The transfer function gives a relation between the input and the output. It can be computed from:
\begin{itemize}
    \item  the mathematical model of the transduction mechanism,
    \item or the experimental characterization of the transduction (approximation of the transfer function).
\end{itemize}

> The span refers to the sensor dynamic range: $DR = 20 \log_{10} \left( \frac{s2}{s1} \right)$ [dB].

Reviewer: `siunitx` permet de typeset les unités :)

Reviewer: Laissons comme ça alors, je n'aurai pas le cours et à part moi tout le monde s'en fiche de toute façon 😛

$$ S = A + Bs $$

> With $s$ the physical input stimulus (speed, pressure, temperature,...), $S$ the output signal, $A$ the intercept/\textbf{offset} (zero-crossing output signal) and $B$ the \textbf{sensitivity}/gain (slope).\textbf{ A and B are determined by calibration} (two parameters to determine).

Reviewer: `\ldots`, dans le `\textbf{}` il faudrait mettre `$A$` et `$B$`.

\subsubsection{Use linear piecewise approximation (LPA)}

Using a simple mapping of the nodes present on the transfer function, the microcontroller determines 
on which interval the measured data is located, and applies a simple rule of three.

\begin{itemize}
    \item Light computation
    \item Scalable memory space requirement (only the node positions are stored)
    \item Trade-off between memory space and accuracy
\end{itemize}

\subsubsection{Iterative computation}

After a first guess of the solution $s_0$, a new value $s_{i+1}$ is determined 
following the Newton-Raphson's formula: 

> $$s_{i+1} = s_{i} - \frac{f(s_i)-S_i}{f'(s_i)}$$

Reviewer: `\[ \]` au lieu de `$$ $$`

\end{figure}

Inaccuracy is measured as the highest deviation of a value represented by the sensor from the 
ideal or true value at its input. There are three categories of inaccuracy :

\begin{itemize}
    \item \textbf{Systematic} for all sensors
that leads to an approximate transfer function
    \item \textbf{Stochastic} from one fabricated
sensor to another (different readings on different sensors in the same conditions)

>     \item \textbf{Stochastic in time} due to the noise (with a single senor)

Reviewer: sensor*

Author: ```suggestion
    \item \textbf{Stochastic in time} due to the noise (with a single sensor)
```

\begin{minipage}[c]{.5\linewidth}	  
A hysteresis error is a deviation of the output of the sensor at a specified point of the input signal when it is approached from the opposite directions.

> Typical causes for hysteresis are friction and structural changes in the materials. It is a typical characteristics of mechanical sensors. 

Reviewer: characteristic*

Author: ```suggestion
Typical causes for hysteresis are friction and structural changes in the materials. It is a typical characteristic of mechanical sensors. 
```

\subsubsection{Saturation and Dead-band}

\begin{minipage}[c]{.45\linewidth}	  
\textbf{Saturation} : When the sensor reaches
its operating limits (e.g. output voltage saturates at $V_{DD}$).
\begin{figure}[H]
    \centering
    \includegraphics[width = 0.7\textwidth]{L1/img/sat.PNG}
\end{figure}
\end{minipage} \hfill
\begin{minipage}[c]{.45\linewidth}
\textbf{Dead-band} : Output signal stays at a
certain value (often zero) for a range of input stimuli
\begin{figure}[H]
    \centering
    \includegraphics[width = 0.7\textwidth]{L1/img/dead-band.PNG}
\end{figure}
\end{minipage}
   

> Both contribute to the non-linearity of the sensor if a linear transfer function is assumed !

Reviewer: nonlinearity*

Author: ```suggestion
Both contribute to the nonlinearity of the sensor if a linear transfer function is assumed !
```

\end{figure}

\begin{itemize}

> \item Output-voltage sensors require a low series $Z_{out}$ to have a small loss of voltage across it.

Reviewer: Un peu lourd, mais `$Z_{\textnormal{out}}$` serait plus beau...

Author: ```suggestion
\item Output-voltage sensors require a low series $Z_{\textnormal{out}}$ to have a small loss of voltage across it.
```

>  \item Output-current sensors require a high parallel $Z_{out}$ to have a small current going in it.

Reviewer: Même remarque que 306.

Author: ```suggestion
 \item Output-current sensors require a high parallel $Z_{\textnormal{out}}$ to have a small current going in it.
```

>  \item $Z_{out}$ may contain $R$, $L$, $C$ components.

Reviewer: cf. 306.

Author: ```suggestion
 \item $Z_{\textnormal{out}}$ may contain $R$, $L$, $C$ components.
```

For example, thermal sensors typically have a long “warm-up time” $\rightarrow$ slow response.
Dynamic characteristics can be classified by the order of the sensor:
\begin{itemize}
\item Zero-order: time-independent response $\rightarrow$ responds instantaneously
\item First-order: the relationship between $s(t)$ and $S(t)$ is
described by a first-order differential equation
\item Second-order: the relationship between $s(t)$ and $S(t)$ is
described by a second-order differential equation
\item \dots
\end{itemize}

\paragraph{First-order response}
Characterized by a cut-off frequency in the frequency domain
and a time constant in the time domain. Typical of sensors with one energy storage component
(e.g. a thermal capacity, an electrical capacitor, \dots).

\begin{figure}[H]
    \centering
    \includegraphics[width = 0.6\textwidth]{L1/img/first-order.PNG}
\end{figure}

> \paragraph{Second-ordrer response}

Reviewer: Second-order*

Author: ```suggestion
\paragraph{Second-order response}
```
